bin/datacenter-generator.ts

```diff
@@ -88,6 +88,8 @@ if (type_schema.definitions.DatacenterSchema.anyOf) {
 }
 
 await Deno.writeTextFile(path.join(datacenters_dir, './datacenter.schema.json'), JSON.stringify(type_schema, null, 2));
+await Deno.writeTextFile(path.join(datacenters_dir, './datacenter-schema.ts'), `export default ${JSON.stringify(type_schema, null, 2)}`);
```

Comment:
What is this used for? Couldn't things just import the json file if they want?

Reply:
They can, but the problem comes when you compile the application. JSON files do not get included and honestly for our use case there is no reason for it to be a json file outside of external schema validation tools. Our choices were to try and dynamically package the files up somehow or do something like this. I figured this was cleaner.